README.md

```diff
@@ -89,40 +77,38 @@ Note: The ChromeDriver version must match the Chrome browser version on your mac
     set BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
     ```
 
-    Alternatively, you can also hardcode username and access_key objects in the [single.json](src/resources/single.json) file, [parallel.json](src/resources/parallel.json) and [local.json](src/resources/local.json).
-
-Note:
-
--   The exact test capability values can be easily identified using the [Browserstack Capability Generator](https://browserstack.com/automate/capabilities)
+    Alternatively, you can also hardcode username and access_key objects in the browserstack.yml file, 
 
 ## Running Your Tests
 
-### Run the entire test suite in parallel on a single BrowserStack browser
+### Run the entire test suite in parallel on a BrowserStack browser
 
-In this section, we will run the tests in parallel on a single browser on Browserstack. 
+In this section, we will run the tests in parallel on a browser on Browserstack. 
 
 -   How to run the test?
 
-    To run the entire test suite in parallel on a single BrowserStack browser, use the following command:
+    To run the entire test suite in parallel on a BrowserStack browser, update the 'platforms' in the browserstack.yml file &
+    
+    use the following command:
 
     ```sh
-    paver run single remote
+    browserstack-sdk pytest -s src/test/suites/*.py
     ```
 
 -   Output
 
-    This run profile executes the entire test suite in parallel on a single BrowserStack browser. Please refer to your [BrowserStack dashboard](https://automate.browserstack.com/) for test results.
+    This run profile executes the entire test suite in parallel on a BrowserStack browser. Please refer to your [BrowserStack dashboard](https://automate.browserstack.com/) for test results.
 
 ### Run the entire test suite in parallel on multiple BrowserStack browsers
 
 In this section, we will run the tests in parallel on multiple browsers on Browserstack.
 
 -   How to run the test?
 
-    To run the entire test suite in parallel on multiple BrowserStack browsers, use the following command:
+    To run the entire test suite in parallel on multiple BrowserStack browsers, update the 'parallelsperplatform' in the browserstack.yml file & use the following command:
```

Comment:
Please create multiple YML files for each config and use BROWSERSTACK_CONFIG_FILE  env variable to run different configs